database_cleanup/models/purge_modules.py

```diff
@@ -67,6 +67,8 @@ def find(self):
         for module in IrModule.search([('to_buy', '=', False)]):
             if get_module_path(module.name, display_warning=False):
                 continue
+            if module.name == 'studio_customization':
```

Comment:
Can't we add this directly on domain ?

(or atleast combine it with` if get_module_path(module.name, display_warning=False)` above)

Reply:
adding into domain the negative condition is also an option.

Reply:
Changes done